java-tools/OsmAndServer/src/main/resources/application.yml

```diff
@@ -111,6 +116,9 @@ spring:
     username:
     password:
     driverClassname:
+  searchtestdatasource:
+    url: jdbc:sqlite:${osmand.files.location}/search_test.db
```

Comment:
better to have variable like url:dbc:sqlite:${SEARCH_TEST_DB:...}

As it single database then we should store it somewhere /home/search-db/search_test.db.